examples/managementCli/src/index.ts

```diff
@@ -0,0 +1,496 @@
+  .option(
+    '-t, --tenants <t1,t2>',
+    `Access key's tenant IDs`,
+    (val: string, memo: string[]) => {
+      memo.push(val);
+      return memo;
+    },
+    [],
+  )
+  .action(async (name, expireTime, options) => {
+    handleSdkRes(
+      await sdk.management.accessKey.create(
+        name,
+        expireTime,
+        undefined,
+        options.tenants?.map((tenantId: string) => ({ tenantId })),
+      ),
+    );
+  });
+
+// access-key-update
+program
+  .command('access-key-update')
+  .description('Update an access key')
+  .argument('<id>', 'Access key ID')
+  .argument('<name>', 'Access key name')
+  .action(async (id, name) => {
+    handleSdkRes(await sdk.management.accessKey.update(id, name));
+  });
+
+// access-key-delete
+program
+  .command('access-key-delete')
+  .description('Delete an access key')
+  .argument('<id>', 'Access key ID')
+  .action(async (id) => {
+    handleSdkRes(await sdk.management.accessKey.delete(id));
+  });
+
+// access-key-load
+program
+  .command('access-key-load')
+  .description('Load an access key')
+  .argument('<id>', 'Access key ID')
+  .action(async (id) => {
+    handleSdkRes(await sdk.management.accessKey.load(id));
+  });
+
+// access-key-search-all
+program
+  .command('access-key-search-all')
+  .description('Search for all access keys')
+  .action(async () => {
+    handleSdkRes(await sdk.management.accessKey.searchAll());
+  });
+
+// access-key-deactivate
+program
+  .command('access-key-deactivate')
+  .description('Deactivate an access key')
+  .argument('<id>', 'Access key ID')
+  .action(async (id) => {
+    handleSdkRes(await sdk.management.accessKey.deactivate(id));
+  });
+
+// access-key-activate
+program
+  .command('access-key-activate')
+  .description('Activate an access key')
+  .argument('<id>', 'Access key ID')
+  .action(async (id) => {
+    handleSdkRes(await sdk.management.accessKey.activate(id));
+  });
+
+// *** Tenant commands ***
+
+// tenant-create
+program
+  .command('tenant-create')
+  .description('Create a new tenant')
+  .argument('<name>', 'Tenant name')
+  .action(async (name) => {
+    handleSdkRes(await sdk.management.tenant.create(name));
+  });
+
+// tenant-update
+program
+  .command('tenant-update')
+  .description('Update a tenant')
+  .argument('<id>', 'Tenant ID')
+  .argument('<name>', 'Tenant name')
+  .action(async (id, name) => {
+    handleSdkRes(await sdk.management.tenant.update(id, name));
+  });
+
+// tenant-delete
+program
+  .command('tenant-delete')
+  .description('Delete a tenant')
+  .argument('<id>', 'Tenant ID')
+  .action(async (id) => {
+    handleSdkRes(await sdk.management.tenant.delete(id));
+  });
+
+// tenant-all
+program
+  .command('tenant-all')
+  .description('Load for all tenants')
+  .action(async () => {
+    handleSdkRes(await sdk.management.tenant.loadAll());
+  });
+
+// *** Group commands ***
+
+// group-all-for-tenant
+program
+  .command('group-all-for-tenant')
```

Comment:
Missing from readme, can we generate the docs somehow? or have usage command

Reply:
there is an help command which also appears in the readme
![image](https://github.com/descope/node-sdk/assets/10514677/4b198b04-7de1-457f-ab6f-dbf967cba9b1)

I don't want to auto-generate the readme from the command, its an overkill imo especially for management cli example

but I will add those commands to readme as well